lib/electric_slide/call_queue.rb

```diff
@@ -219,6 +219,7 @@ def conditionally_return_agent(agent, return_method = @agent_return_method)
         return_agent agent
       else
         logger.debug "Not returning agent #{agent.inspect} to the queue"
+        return_agent agent, :after_call
```

Comment:
This change introduces a new state to ElectricSlide, `:after_call`. It needs to be documented on the `Agent` class and probably the README.  Also, we need to think through whether we want to offer a "mode" to ElectricSlide consumers where agents check themselves back in as `:available` immediately, or wait for manual intervention.